src/main/java/core/basesyntax/strategy/impl/JsonDataParserImpl.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.DataParser;
+import java.util.Collections;
+import java.util.List;
+
+public class JsonDataParserImpl implements DataParser {
+    @Override
+    public List<FruitTransaction> parseData(String data) {
+        //TODO: implement method
```

Comment:
if you don't add logic for such implementations, remove these implementations at all